change/@microsoft-teams-js-564f452b-e760-4a7a-bc80-fbc7288b70fb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Git magic week of 4/18/2022 \n1. IMeetingDetails has been renamed to IMeetingDetailsResponse and IDetails has been replaced with IMeetingOrCallDetailsBase<T>. As such, meeting.getMeetingDetails() now takes in a callback which takes in IMeetingDetailsResponse rather than IMeetingDetails. \n2. pages.returnFocus() now works in any FrameContext rather than just with content.",
```

Comment:
Sorry I didn't notice this before: do you think it's worth commenting on adding `ipados` to the `HostClientType` enum in the change file?